docs/Upgrade.md

```diff
@@ -133,6 +133,144 @@ Here are the available codemods you may need to run on your codebase:
 
 Check out React Query [codemod documentation](https://tanstack.com/query/latest/docs/react/guides/migrating-to-v5#codemod) for more information.
 
+## `<Admin menu>` Is No Longer Supported
+
+The `<Admin menu>` prop was deprecated since 4.0. It's no longer supported. If you want to customize the application menu, you'll have to do it un a custom Layout instead:
```

Comment:
```suggestion
The `<Admin menu>` prop was deprecated since 4.0. It's no longer supported. If you want to customize the application menu, you'll have to do it in a custom Layout instead:
```